README.md

```diff
@@ -1,19 +1,121 @@
 # robotics-prototype
-This repo contains the beginning of the new (2017-2018) robotics software team code
+This repo contains the beginning of the new (2017-2018) Robotics software team code.
 
-## How to upload arduino scripts from the odroid
+## Contributing and Development Environment Instructions
+Firstly, this project is built in Python 3.3+. You need to have a version of Python installed that is 3.3+. Make sure that whenever you use `python` or `python3` or whatever later on meets this requirement.
```

Comment:
GTK code is in 2.7, even though we are moving away from that and python 2 in general

Reply:
@PeterGhimself I was able to run everything using Gtk3 and Python 3+. I guess the question is should we bother putting it in the requirements.txt? I looked into setting up Travis with `apt-get install <package>` directives, it seems possible, might be more trouble than its worth, though.

Reply:
I mean, if it's not too much trouble to figure out (like if those `apt-get install` directives work out), then it's fine for now I guess, since #42 is still under development and not merged in with master yet. If's too much of a hastle to figure out I guess we can omit it from `requirements.txt`.

Reply:
@PeterGhimself Ok so I looked into this ad nauseam (trying to get `PyGObject` and `python3-gi` to work together with Travis-CI).

My conclusion is the latest version of `PyGObject` (3.28.3) requires a version of `python3-gi` too recent for the latest one available on Trusty. This is seen in the screenshot where `gobject-introspection-1.0 >= 1.46.0` is requested by `PyGObject`, but Trusty (i.e. Ubuntu 14.04) only supports up to 1.40.0 as depicted [here](https://launchpad.net/ubuntu/+source/gobject-introspection). In other words, TravisCI would need to offer a Ubuntu version at minimum Xenial (16.04+) in which 1.46.0 would be available (but it doesn't).

![image](https://user-images.githubusercontent.com/6502401/42678327-8d1bca84-864c-11e8-85a9-c7ef16795627.png)

More reason to move on to Flask!!

Reply:
💯 👌 🔥